### DISCUSSION TITLE
DataTable current object

### DISCUSSION THREAD

**Discussion by ashwini-mv** [codeceptjs/CodeceptJS#3205] [Q&A]
I am trying to use DataTable with typescript, I am getting this error -> Property 'current' does not exist on type 'SupportObject'. Can someone help me with this?

const accounts = new DataTable(['login', 'password']); //
accounts.add(['davert', '123456']); // adding records to a table
accounts.add(['admin', '123456']);

Data(accounts).Scenario('Search for users', async({I,current}) => {

**Comment by kobenguyent** [comment]
@ashwini-mv Did you have something like this in your `steps.d.ts`?

```
 /// <reference types='codeceptjs' />
type steps_file = typeof import('./steps_file');

declare namespace CodeceptJS {
  interface SupportObject { I: I, current: any }
  interface Methods extends REST {}
  interface I extends ReturnType<steps_file> {}
  namespace Translation {
    interface Actions {}
  }
}
```

**Reply by ashwini-mv** [reply]
@PeterNgTr  I know that works, but that is not a good solution right?  Here I saw this issue is fixed, but it is not available in the latest version ->
https://codecept.discourse.group/t/help-me-with-datatable/769

**Comment by kobenguyent** [comment]
it is with current version. did you run` codeceptjs def .`?

**Reply by ashwini-mv** [reply]
@PeterNgTr I did, it says
TypeScript Definitions provide autocompletion in Visual Studio Code and other IDEs
Definitions were generated in steps.d.ts
Still I am not able to use 'current' object

**Comment by kobenguyent** [comment]
@ashwini-mv my project is working fine. Which codeceptjs version are you using?

**Reply by ashwini-mv** [reply]
@PeterNgTr I am using    "codeceptjs": "^3.2.2", I am using CodeceptJs with Playwright

**Reply by kobenguyent** [reply]
I'm using pretty much the same. Could you try deleting your local `node_modules` and run `npm i` again?

**Reply by ashwini-mv** [reply]
Sorry if I am confusing you, Just to let you know  with interface SupportObject { I: I, current: any } this works fine although it won't autosuggest.
My question is, Is this possible to use it without mentioning 'current: any' in steps.d.ts file ?
